Azure/sonic-buildimage
# [mux]: Fix `mark_dhcp_packet`

Fixes #9374 

#### Why I did it
Two issues prevent the `mux` systemd service from starting properly:
- The `mux.service` unit file contains two `[Service]` sections, making it invalid
- The `mark_dhcp_packet.py` file is not executable in the image

#### How I did it
- Consolidate the two `[Service]` sections by moving the `ExecStartPre` line for `mark_dhcp_packet.py` to the first section and removing the second.
- Make the `mark_dhcp_packet.py` file executable
- Also clean up `mark_dhcp_packet.py`
    - Remove unused imports
    - Fix spacing and line lengths to conform to PEP8

#### How to verify it
- Ensure that  the `mux` service start properly:
```
admin@sonic:~$ systemctl status mux
● mux.service - MUX Cable Container
     Loaded: loaded (/lib/systemd/system/mux.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2021-11-24 20:16:51 UTC; 5min ago
    Process: 118541 ExecStartPre=/usr/local/bin/write_standby.py (code=exited, status=0/SUCCESS)
    Process: 120521 ExecStartPre=/usr/local/bin/mark_dhcp_packet.py (code=exited, status=0/SUCCESS)
    Process: 120649 ExecStartPre=/usr/bin/mux.sh start (code=exited, status=0/SUCCESS)
   Main PID: 120723 (mux.sh)
      Tasks: 2 (limit: 19193)
     Memory: 18.1M
     CGroup: /system.slice/mux.service
             ├─120723 /bin/bash /usr/bin/mux.sh wait
             └─120724 /usr/bin/python3 /usr/local/bin/container wait mux
```

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
